core/cmdbsource.class.inc.php

```diff
@@ -350,14 +350,20 @@ public static function IsDB($sSource)
 
 	}
 
+	/**
+	 * Get the version of the database server.
+	 *
+	 * @return string
+	 * @throws \MySQLException
+	 */
 	public static function GetDBVersion()
 	{
-		$aVersions = self::QueryToCol('SELECT Version() as version', 'version');
-		return $aVersions[0];
+		return static::QueryToScalar('SELECT VERSION()', 0);
 	}
 
 	/**
-	 * @return string
+	 * @deprecated Use `CMDBSource::GetDBVersion` instead.
+	 * @see mysqli_get_server_info
```

Comment:
```suggestion
	 * @uses mysqli_get_server_info
```

We prefer to write `@uses` in such case (see https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/uses.html#uses-used-by)

Reply:
Thanks, didn't know that one.